Source/GameBaseFramework/Private/Equipment/GBFEquipmentManagerComponent.cpp

```diff
@@ -106,6 +106,43 @@ UGBFEquipmentInstance * FGBFEquipmentList::AddEntry( TSubclassOf< UGBFEquipmentD
     return result;
 }
 
+UGBFEquipmentInstance * FGBFEquipmentList::AddEntryFromExistingActor( UGBFEquipmentInstance * equipment_instance, TSubclassOf< UGBFEquipmentDefinition > equipment_definition )
+{
+    check( equipment_definition != nullptr );
+    check( equipment_instance != nullptr );
+    check( OwnerComponent != nullptr );
+    check( OwnerComponent->GetOwner()->HasAuthority() );
+
+    const auto * equipment_cdo = GetDefault< UGBFEquipmentDefinition >( equipment_definition );
+
+    auto & new_entry = Entries.AddDefaulted_GetRef();
+    new_entry.EquipmentDefinition = equipment_definition;
+    new_entry.Instance = equipment_instance; //@TODO: Using the actor instead of component as the outer due to UE-127172
```

Comment:
our convention for TODO is `:TODO:`
https://github.com/TheEmidee/UECodingStandard/blob/master/cpp_standards.md#cppcomments

Reply:
Done, sorry dumb copy/paste